lib/variable/test/transform_binary_test.cpp

```diff
@@ -372,7 +372,8 @@ void check_binned_with_dense(Variable &binned, const Variable &dense,
   const auto dense_binned = transform<double>(dense, binned, op, name);
 
   for (scipp::index i = 0; i < indices.dims().volume(); ++i) {
-    const auto &[begin, end] = indices.values<index_pair>()[i];
+    const auto values = indices.values<index_pair>();
+    const auto &[begin, end] = values[i];
```

Comment:
This and other similar cases are about avoiding a false-positive lifetime warning from returning references from a temporary `ElementArrayView` (return value of `.values()`).